content/articles/communicating/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Sharing and communicating"
 date: 2018-03-21
-csl: ../inbo.csl
+csl: '/tmp/RtmpnGa7xS/research-institute-for-nature-and-forest.csl'
```

Comment:
the R code in the `index.Rmd` translates to a temporary folder on your pc? Is this going to work on the tutorials website?

Reply:
Yes it's going to work, because index.md is already the result rendered by pandoc (from Rmd to md). Several parts of the YAML header are only relevant for pandoc (such as this csl key). pandoc is just called by knitr (locally), when rendering the Rmd to md. In other words, this index.md already contains the formatted citations and bibliography. The site generator just knows how to turn this markdown into html, and it will use a few fields from YAML (such as title) to set up the website. For this reason, we explicitly tell pandoc to return the YAML header in the markdown result (preserve_yaml: true). Obviously the csl part is junk at this stage :smile: .